apps/main/src/llamalend/PageLlamaMarkets/cells/HealthCell.tsx

```diff
@@ -20,7 +20,7 @@ export const HealthCell = ({ row }: CellContext<LlamaMarket, number>) => {
   ) : (
     <Tooltip
       title={softLiquidation ? 'Liquidation Protection On' : 'Position active'}
-      body={<HealthTooltipContent softLiquidation={softLiquidation} />}
+      body={<HealthTooltipContent softLiquidation={!!softLiquidation} />}
```

Comment:
wasn't softliquidation a boolean? :thinking:

Reply:
`const { health, softLiquidation } = data ?? {}`
means the type of `softLiquidated` is `boolean | undefined`, the prop expects only `boolean`, no `undefined`.